src/pages/index.js

```diff
@@ -85,7 +85,7 @@ export default class Index extends React.Component {
                 <Contact class={this.state.modal.contact} endTransitionHandler={(name) => this.endArticleTransition(name)} />
```

Comment:
In the pdf, could you put the actual url to the google stuff in there? I'm afraid that some browsers will not be allow the pdf to be interactive, thus the link wouldn't be a link